build.sbt

```diff
@@ -1126,3 +1126,9 @@ intellijToSample := {
   } else
     s.log.info("Aborting.")
 }
+
+/** Find a specific module's JAR in a classpath, comparing only organization and name */
+def findJar(files: Seq[Attributed[File]], dep: ModuleID): Option[Attributed[File]] = {
+  def extract(m: ModuleID) = (m.organization, m.name)
+  files.find(_.get(moduleID.key).map(extract _) == Some(extract(dep)))
```

Comment:
`Some(extract(dep))` could be stored in a local instead of re-creating it over again